code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -1355,37 +1357,59 @@ datum
 				if (prob(25))
 					M.reagents.add_reagent("histamine", rand(5,10) * mult)
 				if (our_amt < 20)
-					M.take_toxin_damage(1 * mult)
-					random_brute_damage(M, 1 * mult)
+					M.take_toxin_damage(0.75 * mult)
+					delimb_counter += 0.5 * mult
```

Comment:
```suggestion
```
>6u of venom would cause a guaranteed delimb with this unless purged.

Reply:
changed to a +0.2, you need at least 15 units and 2:30+ minutes for a single limb loss then, if it's still too much, i can just remove it as requested

Reply:
actually, yeah i don't see why it even needs to count towards that in low doses anyway, away that one goes